### DISCUSSION TITLE
为什么youtube双语字幕都是显示的中文

### DISCUSSION THREAD

**Discussion by stevenli11** [immersive-translate/immersive-translate#2029] [Help]
### 概述 | Summary

为什么youtube双语字幕都是显示的中文，我需要一个英文一个中文对照看

**Comment by keyboard3** [comment]
你YouTube字幕选择的是中文，然后你翻译得也是中文。肯定就这是中文，你视频选择英文，目标语言是中文，那出来就是双语，或者反过来

**Reply by stevenli11** [reply]
> 你YouTube字幕选择的是中文，然后你翻译得也是中文。肯定就这是中文，你视频选择英文，目标语言是中文，那出来就是双语，或者反过来

我选的是英文自动生成

**Reply by keyboard3** [reply]
录个屏看下

**Reply by stevenli11** [reply]
> 录个屏看下

我知道原因了，我开了圈X，可能里面有规则自动翻译中文😂
